README.md

```diff
@@ -18,6 +18,8 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 
 1. [CatalogReaderExample](CatalogReaderExample) is a sample showing how to read the NuGet API's catalog resource.
 
+1. [NuGetProtocolSamples](NuGetProtocolSamples) shows how to use the [`NuGet.Protocol`](https://www.nuget.org/packages/NuGet.Protocol) package.
+
 1. [PackageDownloadsExample](PackageDownloadsExample) shows how to get a package's download counts.
```

Comment:
Can we merge the package downloads example into here?

Reply:
We could, but I'd prefer not to:

1. The PackageDownloadsExample app takes a package id argument. This lets the user play around with the sample to make it more fun.
1. The PackageDownloadsExample sample is complicated. I'd like to keep `NuGet.Protocol.Samples` a showcase of well-supported scenarios.